src/jquery.plusmore.js

```diff
@@ -59,20 +59,53 @@
  *    showMore:
  *       Callback function to setup handling the interaction with the "+ X more" item
  *       Defaults to showing the remainder of the list when the "+ X more" item is clicked
+ *
+ *  Events:
+ *     plusmore.showMore:
+ *        By triggering this event, the caller can programatically run the showMore function on the target element.
+ *        Note: when triggering showMore, the function is executed within the context of the classname configured by 
+ *        moreItemsClass
+ *
```

Comment:
Adding a note to the developer, that when implementing custom formatList, to trigger showMore on the element defined by `moreItemsClass`